hack/testing/test-fluentd-forward.sh

```diff
@@ -1,203 +1,5 @@
-#! /bin/bash
+#!/bin/bash
```

Comment:
Yes.  We can drop it, and everything else in hack/testing, once everything is ported to /test.